### PR TITLE
SEE ALL REMINDERS - RENDERING SUCCESSFULLY

### LINKED ISSUE #1: See All Reminders
When the user visits the root of the application (e.g. the "/" URL), they should see a list of all of the reminders on the page.

- Each reminder should have a class of .spec-reminder-item.
- There is a failing test set up in tests/acceptance/reminder-list-test.js.

Hot Tip: If you put “Closes #1” (assuming the issue is marked #1) in the pull request, your commit, or anywhere in the body of your PR, it will automatically close this issue when it successfully merged into master.

### PR DESCRIPTION
**Pull request by dshendrickson**
All five reminders render to page and have a class of .spec-reminder-item.

pasted image at 2016_12_07 01_23 pm

Issue: closes #1

### CODE REVIEW

File: app/styles/app.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.spec-reminder-item {
+  
+}
```
Comment (martensonbj) [inline app/styles/app.css]: If you don't have anything you want to style yet lets get rid of unnecessary code.

Comment (andrewLcrist) [inline app/styles/app.css]: Sounds good. We will update the PR and resubmit.

**Comment by dshendrickson** [comment]
CSS changes as per your request.

**Comment by martensonbj** [comment]
Let's also write an acceptance test to cover the functionality for this issue.

**Comment by martensonbj** [comment]
GREAT WORK TEAM - ALSO CONVENTION IS TO WRITE PR REQUEST TITLES NOT IN ALL CAPS. 👍  Sorry for the confusion.

**Pull request merged**